models/articles.models.js

```diff
@@ -98,3 +99,38 @@ exports.insertCommentsByArticleID = ({username, body}, article_id)=>{
       
         
     }
+
+
+    
+exports.updateArticle = async (voteChange, article_id)=>{
+    let inc_votes = voteChange.inc_votes;
+    const article_id_num = Number(article_id)
+    let currentVotes = 0
+ 
+    articles = await articlesTable()
+
+   
+    articles.forEach((article)=>{
+      
+    
+        if(article.article_id === article_id_num){
+            inc_votes+=article.votes
+        }
+        
+    })
+
+    return db.query(`
+    UPDATE articles
+    SET votes = $1
```

Comment:
in your SQL string you can increment votes by doing votes = votes + $1.  There is no need to query your db to get the current votes..  I would change this!